content/en/entities/StatusEdit.md

```diff
@@ -78,7 +78,7 @@ aliases: [
 ### `account` {#account}
 
 **Description:** The account that published this revision.\
-**Type:** Account\
+**Type:** [Account]\
```

Comment:
`[Account]({{<relref "entities/Account">}})` perhaps?

Reply:
yes, you are right, fixed.